consensus/spos/bls/errors.go

```diff
@@ -4,3 +4,13 @@ import "errors"
 
 // ErrNilSentSignatureTracker defines the error for setting a nil SentSignatureTracker
 var ErrNilSentSignatureTracker = errors.New("nil sent signature tracker")
+
+var JobIsNotDoneError error = errors.New("job is not done")
```

Comment:
Please follow the model of errors names from the file
e.g all error should start with `Err`

also need comment on exported errors